administrator/language/de-DE/de-DE.mod_latestactions.ini

```diff
@@ -4,9 +4,9 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-MOD_LATESTACTIONS="Action Logs - Latest"
-MOD_LATESTACTIONS_FIELD_COUNT_LABEL="Count"
-MOD_LATESTACTIONS_FIELD_COUNT_DESC="The number of items to display (default 5)."
-MOD_LATESTACTIONS_LAYOUT_DEFAULT="Default"
-MOD_LATEST_ACTIONS_NO_MATCHING_RESULTS="No Matching Results"
-MOD_LATESTACTIONS_XML_DESCRIPTION="This module shows a list of the most recent actions."
+MOD_LATESTACTIONS="Aktivitäten - Neueste"
+MOD_LATESTACTIONS_FIELD_COUNT_LABEL="Anzahl"
+MOD_LATESTACTIONS_FIELD_COUNT_DESC="Die Anzahl der Einträge die angezeigt werden soll (Standard 5)"
```

Comment:
> Die Anzahl der Einträge die angezeigt werden sollen (Standard 5)

sollen vs soll?

Reply:
sollen (ist ja Mz)

Reply:
Ist halt so schon mal drin in einer anderen Datei
https://github.com/joomlagerman/joomla/blob/abb7e60f45acd3f104904d9738de0d016efd9ff1/administrator/language/de-DE/de-DE.mod_logged.ini#L11

Notfalls beide anpassen oder erstmal mergen.

Reply:
Nope :-P

https://github.com/joomlagerman/joomla/commit/626441c0acfa5b39dea4e7a9bcb95b83fefb6ed1